### PR TITLE
#1801 - Reorganize Workflow Folders

### PR DESCRIPTION
**Pull request by katrina-aot**
Update Camunda workflows to version 8, removing version 7 files

### CODE REVIEW

File: sources/packages/backend/workflow/src/deploy.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ import {
   console.info(`**** Deploying to Camunda ****\n`);
   console.info(`Deploying to Zeebe address ${process.env.ZEEBE_ADDRESS}`);
 
-  const directory = path.resolve(__dirname, `../BPMN/camunda-8`);
+  const directory = path.resolve(__dirname, `../src/workflow-definitions`);
```
Comment (andrewsignori-aot) [inline sources/packages/backend/workflow/src/deploy.ts:26]: Can you please use the path below?
```ts
path.resolve(__dirname, "./workflow-definitions")
```

**Review by andrewsignori-aot — COMMENTED** [review]
Thanks so much for the cleanup. Just one minor comment, please take a look.

File: sources/packages/backend/workflow/src/deploy.ts (hunk 1 above)
Comment (katrina-aot) [reply sources/packages/backend/workflow/src/deploy.ts]: updated

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for making the change, looks good 👍

**Review by lewischen-aot — APPROVED** [review]
Thanks for making the change. LGTM 👍

**Review by sh16011993 — APPROVED** [review]
Nice work @katrina-aot 👍 LGTM

**Pull request merged**